packages/vite-environment-provider-cloudflare/src/index.ts

```diff
@@ -23,8 +23,8 @@ import {
 } from 'miniflare';
 
 import * as debugDumps from './debug-dumps';
-import { collectModuleInfo } from './moduleUtils';
-import { readFile, stat } from 'node:fs/promises';
+import { URL } from 'url';
```

Comment:
This should be available on the global object.

```suggestion
```

Reply:
oops, I thought I removed all the unused imports! thanks! 🙂